packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -109,7 +110,10 @@ const AuthenticatedApp = () => {
       userDisplayName={userDisplayName}
     >
       <RBACProvider permissions={permissions} refetchPermissions={refetch}>
-        <PluginsInitializer />
+        <>
+          <NpsSurvey />
+          <PluginsInitializer />
+        </>
```

Comment:
Why does it need a fragment? 🤔

Reply:
I was getting a warning about RBACProvider expecting a single child

Reply:
ahh, that's dumb, ah well! :)